stats/lib/stats-fetchers.js

```diff
@@ -37,14 +37,16 @@ export const fetchDailyDealStats = async (pgPools, filter) => {
   const { rows } = await pgPools.evaluate.query(`
     SELECT
       day::text,
-      tested,
-      index_majority_found AS "indexMajorityFound",
-      indexed,
-      indexed_http AS "indexedHttp",
-      retrieval_majority_found AS "retrievalMajorityFound",
-      retrievable
+      SUM(tested) AS tested,
+      SUM(index_majority_found) AS "indexMajorityFound",
+      SUM(indexed) AS indexed,
+      SUM(indexed_http) AS "indexedHttp",
+      SUM(retrieval_majority_found) AS "retrievalMajorityFound",
+      SUM(retrievable) AS retrievable
     FROM daily_deals
-    WHERE day >= $1 AND day <= $2
+    WHERE day >= date_trunc('day', $1::DATE)
+      AND day <= date_trunc('day', $2::DATE)
```

Comment:
are these two lines also covered by tests?

Reply:
I am not sure, to be honest. I found it in the other query reading from `daily_deals`. On the other thought, I think it does not make sense to call `date_trunc('day')` on a value that already has day-level precision - I'll revert.

Reply:
See 729c1bd